server/appointment/upload-documents/uploadDocuments.php

```diff
@@ -197,10 +211,50 @@ function markAppointmentAsReady($token, $firstName, $lastName, $emailAddress, $p
 	echo json_encode($response);
 }
 
+function storeConsent($reviewConsent, $virtualConsent, $signature, $appointmentId) {
```

Comment:
Small complication here... As this code is written, anyone could CURL this endpoint with any random `$appointmentId` and we'd put it in the DB. To prevent against attacks like this, every other function in this file requires the token/firstName/lastName/phone/email and re-checks that information before doing the action requested. We need to do that same thing here